docs/en/customization/build.md

```diff
@@ -9,20 +9,18 @@ for their particular project.
 - forking existing ones to make changes,
 - or building a lightweight server with only the necessary plugins.
 
-To facilitate this process, we have developed the **Velox** tool, which enables developers to build a RoadRunner server
-binary based on a configuration file specifying the required plugins and repositories for a specific project.
+We created a tool called **Velox** that lets developers build a RoadRunner server binary. They do this by using a
```

Comment:
They? I guess it's better to use: `Velox uses a configuration file that contains the list of plugins that are available in GitHub or GitLab repositories`